subsys/net/lib/nrf_cloud/src/nrf_cloud_transport.c

```diff
@@ -236,19 +236,19 @@ static bool strings_compare(const char *s1, const char *s2, uint32_t s1_len,
 }
 
 /* Verify if the RX topic is a control channel topic or not. */
-static bool nrf_cloud_cc_rx_topic_decode(const struct mqtt_topic *topic, enum nct_cc_opcode *opcode)
+static bool nrf_cloud_cc_rx_topic_dec(const struct mqtt_topic *topic, enum nrfct_cc_opcode *opcode)
```

Comment:
Why this change? Is it related to max line length?

Reply:
yes, it is due to max line